docs/section23/Rule23-13.md

```diff
@@ -0,0 +1,46 @@
+
+# Airside System - Rule 23-13 
+
+**Schema Version:** 0.0.34  
+**Mandatory Rule:** False  
+**Rule ID:** 23-13  
+**Rule Description:** If proposed design includes humidistatic controls then the baseline shall use mechanical cooling for dehumidification and shall reheat to avoid overcooling.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 23 Air-side  
+**90.1 Section Reference:** G3.1.3.18 Dehumidification (Systems 3 through 8 and 11, 12, and 13)  
+**Data Lookup:** None  
+**Evaluation Context:** HVAC system  
+
+**Applicability Checks:**  
+
+1. P-RMD is modeled with at least one air-side system that includes humidistatic controls.  
+
+**Function Calls:**  
+
+1. get_dict_of_zones_and_terminal_units_served_by_hvac_sys
+
+
+**Applicability Checks:**  
+- get a list of the hvac systems and the zones they served in the P_RMR: `hvac_systems_and_zones_p = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(P_RMR)`
+- look through each hvac system in the proposed: `for hvac_p_id in hvac_systems_and_zones_p:`
+    - get the proposed hvac sytem: `hvac_p = get_object_by_id(hvac_p_id)`
+    - set a boolean has_humidifcation_controls equal to false: `has_humidifcation_controls = FALSE`
+    - get the heating system: `heating_system_p = hvac_p.heating_system`
+    - check to see if the heating system has a humidification type other than NONE, if so, there are humidification controls, set has_humidifcation_controls to TRUE: `if heating_system_p.humidification_type != NONE: has_humidifcation_controls = TRUE`
```

Comment:
I am not clear why we are checking for humification here and at the zone level if the rule applies to dehumification